apps/annotation/opencv_annotation.cpp

```diff
@@ -229,21 +229,29 @@ int main( int argc, const char** argv )
         "{ help h usage ? |      | show this message }"
         "{ images i       |      | (required) path to image folder [example - /data/testimages/] }"
         "{ annotations a  |      | (required) path to annotations txt file [example - /data/annotations.txt] }"
+        "{ maxWindowHeight m  |  -1   | (optional) images larger in height than this value will be scaled down }"
+        "{ resizeFactor r  |  2  | (optional) factor for scaling down [default = half the size] }"
     );
     // Read in the input arguments
     if (parser.has("help")){
         parser.printMessage();
         cerr << "TIP: Use absolute paths to avoid any problems with the software!" << endl;
         return 0;
     }
-    string image_folder(parser.get<string>("images"));
-    string annotations_file(parser.get<string>("annotations"));
+    string image_folder = parser.get<string>("images");
```

Comment:
In the case of `std::string` it doesn't make a difference. But that's not true for the general case. In the general case, the two initializations can call different functions. One may compile, while the other not. Or worse yet, it may compile, and differ in (runtime) behaviour. That's why I asked. Not because I thought in this particular case it makes a difference either way, but to see we're aware of this..

For example, 

    struct foo;
    struct bar {
        bar(int);
        explicit bar(double);
        explicit bar(foo);
    };

    // ... {
        bar b0(42);  // bar(int)
        bar b1 = 42;  // bar(int)
        foo f;
        bar b2(f);  // bar(foo)
        // bar b3 = f;  // compile error: bar(foo) is explicit
        bar b4(42.0);  // bar(double)
        bar b5 = 42.0;  // bar(int)

The last line, with `b5`, might surprise some. So might the line with `b3`, although at least the compiler would catch that one..